ImperatorToCK3/Imperator/Countries/CoutryName.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System.Collections.Generic;
```

Comment:
CoutryName.cs -> CountryName.cs

Reply:
Good catch, I fixed the class name but forgot to rename the file.